test/utils/TestUtils.ts

```diff
@@ -0,0 +1,81 @@
+export function bytes32(n: number): string {
+    return '0x' + n.toString().repeat(64).slice(0, 64);
+}
```

Comment:
replace with `web3.utils.toHex`

Reply:
I don't think they do exactly the same. This function actually repeat the same number 64 times and it returns it with `0x` as prefix, while `web3.utils.toHex` actually returns its hexadecimal value. Am I missing anything?

Reply:
well, that's the bit I don't understand. Why does it need to do that? In any case, the intent should be captured in the name of the function.

Reply:
Yes, I've tried to rename it to make it clear what it does

Reply:
I've made a change suggestion in a different comment.